provision/dev/README.md

```diff
@@ -28,11 +28,12 @@ helmfile apply --skip-diff-on-install
 
 ```bash
 ./create_cluster.sh multi
-helmfile apply -e control --skip-diff-on-install
+helmfile apply -e control -l app!=fake-gpu-operator --skip-diff-on-install
 
 # Please set the endpoint address to http://localhost/v1
 llma auth login
 export REGISTRATION_KEY=$(llma admin clusters register worker-cluster | sed -n 's/.*Registration Key: "\([^"]*\)".*/\1/p')
+helmfile apply -e worker -l app=fake-gpu-operator --skip-diff-on-install
 helmfile apply -e worker -l app=llmariner --skip-diff-on-install
```

Comment:
```suggestion
helmfile apply -e worker -l app=fake-gpu-operator -l app=llmariner --skip-diff-on-install
```

or add worker selector

Reply:
Will add a worker selector. I think the conditions are AND, not OR.

```
 Only run using the releases that match labels. Labels can take the form of foo=bar or foo!=bar.
  A release must match all labels in a group in order to be used. Multiple groups can be specified at once.
  "--selector tier=frontend,tier!=proxy --selector tier=backend" will match all frontend, non-proxy releases AND all backend releases.
   The name of a release can be used as a label: "--selector name=myrelease"
```

Reply:
>   "--selector tier=frontend,tier!=proxy --selector tier=backend" will match all frontend, non-proxy releases AND all backend releases.

It seems the multiple selector is OR, and the comma-separated value is AND.

Reply:
ah i think you're right. fixed.